components/bpmn/org.wso2.carbon.bpmn.uuf.ui.stub/src/main/java/org.wso2.carbon.bpmn.uuf.ui/service/BPMNExplorerServiceImpl.java

```diff
@@ -0,0 +1,522 @@
+            response = client.execute(httpPost);
+            rd = new BufferedReader(
+                    new InputStreamReader(response.getEntity().getContent(), Charset.defaultCharset()));
+            StringBuffer imgResult = new StringBuffer();
+            while ((line = rd.readLine()) != null) {
+                imgResult.append(line);
+            }
+            String imageData = imgResult.toString().split(":return")[1];
+            imageUrl = "data:image/png;base64," + imageData.substring(1, imageData.length() - 4);
+
+        } catch (ClientProtocolException e) {
+            log.error("Exception : ", e);
+        } catch (IOException e) {
+            log.error("Exception : ", e);
+        }
+        return imageUrl;
+    }
+
+    /**
+     * Initialization of creating process instances
+     *
+     * @param processId
+     * @param host
+     * @param port
+     */
+    @Override
+    public JSONObject getFormData(String processId, String host, int port) throws JSONException, IOException {
+        HttpGet httpGet = null;
+        CloseableHttpResponse response = null;
+        BufferedReader rd = null;
+        String line;
+        JSONObject rsltObj;
+        JSONObject rtrnObj = new JSONObject();
+
+        setHttpClient(host, port);
+        String processDefId = processId;
+
+        try {
+            httpGet = new HttpGet(baseUrl + "/bpmn/process-definition/" + processDefId + "/properties");
+            httpGet.addHeader("Content-type", "application/json");
+            httpGet.addHeader("Authorization", "Basic YWRtaW46YWRtaW4=");
+
+            response = client.execute(httpGet);
+            rd = new BufferedReader(new InputStreamReader(response.getEntity().getContent(), Charset.defaultCharset()));
+            StringBuffer rsltForm = new StringBuffer();
+            while ((line = rd.readLine()) != null) {
+                rsltForm.append(line);
+            }
+
+            rsltObj = new JSONObject(rsltForm.toString());
+            // When error occurs due to invalid data in the process creation
+            if (!rsltObj.has("data") && rsltObj.has("errorMessage")) {
+                rtrnObj.put("error-message", rsltObj.get("errorMessage"));
+            } else {
+                JSONArray formData = new JSONObject(rsltForm.toString()).getJSONArray("data");
+                // When form data is not required for process creation
+                if (formData.length() <= 0) {
+                    rtrnObj = createProcessInstance(processDefId);
+                }
+                // When it is required to fill a form to create process instance
+                else {
+                    rtrnObj.put("processDefId", processDefId);
+                    rtrnObj.put("formData", formData);
+                }
+            }
+        } catch (ClientProtocolException e) {
+            log.error("Exception : ", e);
+        } catch (IOException e) {
+            log.error("Exception : ", e);
+        } finally {
```

Comment:
use multi catch to catch all exceptions in a single catch as catch(ClientProtocolException | IOException e)